pom.xml

```diff
@@ -63,7 +63,7 @@
         <gravitee-integration-api.version>5.1.0</gravitee-integration-api.version>
         <gravitee-json-validation.version>2.0.1</gravitee-json-validation.version>
         <gravitee-kubernetes.version>3.7.0</gravitee-kubernetes.version>
-        <gravitee-node.version>7.11.4</gravitee-node.version>
+        <gravitee-node.version>7.15.0</gravitee-node.version>
```

Comment:
I've been trapped multiple times because of gravitee-node being upgraded. Could you look into Helm test to show the version of the plugins coming from gravitee-node and that can be downloaded at startup? (ex:  deployment_federation_test.yaml, values.yaml)